index.html

```diff
@@ -0,0 +1,1356 @@
+                    </button>
+                </div>
+            </div>
+            <nav>
+                <ul>
+                    <li><a href="#">U.S.</a></li>
+                    <li><a href="#">World</a></li>
+                    <li><a href="#">Business</a></li>
+                    <li><a href="#">Tech & Science</a></li>
+                    <li><a href="#">Culture</a></li>
+                    <li><a href="#">Newsgeek</a></li>
+                    <li><a href="#">Sport</a></li>
+                    <li><a href="#">Healt</a></li>
+                    <li><a href="#">The Debate</a></li>
+                    <li><a href="#">Vantage</a></li>
+                    <li class="last-nav-link"><a href="#">Weather</a></li>
+                    <li class="search-bar"><input type="text" placeholder="Search"></li>
+                </ul>
+            </nav>
+        </header>
+        <main class="container">
+            <div class="row">
+                <section class="top-story-section col-sm-12 col-md-7 order-md-2 col-lg-5 order-lg-2">
+                    <div class="sticky">
+                        <div class="section-title">
+                            <h2><a href="#">Top story</a></h2>
+                        </div>
+                        <article class="story ">
+                            <div class="story-img-container">
+                                <div class="story-img-category">
+                                    <a href="#">u.s.</a>
+                                </div>
+                                <div class="story-img">
+                                    <a href="#">
+                                        <img src="images/mcconnell-stimulus.jpg" alt="crowded street">
+                                    </a>
+                                </div>
+                            </div>
+                            <div class="story-content">
+                                <h2><a href="#">New Stimulus Could Come in 'Next Month or So,' McConnell Says</a></h2>
+                                <div class="summary">
+                                    <p>The Kentucky Republican has said in recent weeks that he is eyeing another stimulus worth around $1 trillion that likely will not include another round of checks for Americans.</p>
+                                </div>
+                            </div>
+                        </article>
+                    </div>
```

Comment:
- [ ] I am getting more margin than expected in here. Kindly fix that. I am getting the same issue for small screens. Instead of having a container, I think you can have a container fluid and set a max-width. I think that's the way they achieve it. You can check that!

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83210784-5b8a3b80-a121-11ea-9660-6cc2d350c984.png) | ![image](https://user-images.githubusercontent.com/57414789/83210751-457c7b00-a121-11ea-923a-7434e3fa07cd.png)